Cargo.toml

```diff
@@ -107,8 +107,12 @@ full = [
     "indexmap",
     "eyre",
     "anyhow",
+    "pyo3",
 ]
 
+# This makes `#[cfg_attr(feature = "pyo3", pyclass)]` in our own tests work, it has no other function
+pyo3 = []
```

Comment:
I wonder... should we move this to the `pytests` crate or something similar, so that we don't need to expose a no-op feature to the users?

To avoid other code changes, you _might_ also just be able to hack it by adding `RUSTFLAGS=--cfg "feature=pyo3"`, which is all `cargo` really does under the hood I think?

Reply:
so like moving the ui test to `pytests/src`? The `RUSTFLAGS` would also do, but imho it'd be nice if `cargo test` would continue to work.

I've also been thinking about using a different feature name for the tests, but i couldn't come up with a good design for that either, especially for cases such as the guide doc test